packages/server/lib/server.ts

```diff
@@ -51,7 +51,7 @@ import {
 import oAuthSessionService from './services/oauth-session.service.js';
 import { deleteOldActivityLogs } from './jobs/index.js';
 import migrate from './utils/migrate.js';
-import tracer from './apm.js';
+import tracer from './tracer.js';
```

Comment:
@bodinsamuel this was added so it is imported twice. Any reason to not consolidate this into one import at the top of the file?

Reply:
no reason indeed, the first import is there for instrumentation it should stay on top but you can merge 👍🏻